packages/features/eventtypes/components/Locations.tsx

```diff
@@ -22,6 +26,22 @@ export type TLocationOptions = Pick<EventTypeSetupProps, "locationOptions">["loc
 export type TDestinationCalendar = { integration: string } | null;
 export type TPrefillLocation = { credentialId?: number; type: string };
 
+type LocationInputCustomClassnames = {
+  organizerAddressInputClassname?: string;
+  organizerPhoneInputClassName?: string;
+};
+
+export type LocationCustomClassNames = {
+  locationSelectClassnames: locationSelectCustomClassnames;
+  removeLocationButton?: string;
+  removeLocationIcon?: string;
+  addLocationButton?: string;
+  organizerInputClassnames?: {
```

Comment:
I see that some of the keys have `Classnames` at the end and some don't. It would be good if either both have `Classnames` or none I think.

My vote would go for not having `Classnames` suffix because they already are passed via the `customClassnames` object. Maybe it's worth checking how do other UI libraries out there do this.

Reply:
Here’s the style I followed: 

- If it’s an object that contains multiple classnames, it ends with a `Classnames` suffix.
- If it’s an actual classname, we don’t add the `Classname` suffix.